src/azul/indexer/transform.py

```diff
@@ -68,3 +71,12 @@ def get_aggregator(cls, entity_type) -> EntityAggregator:
         entities of types other than X.
         """
         raise NotImplementedError
+
+    @classmethod
+    @abstractmethod
```

Comment:
As I believe we discussed, this doesn't need to be abstract. The default implementation of this method should simply do nothing.